platform/admin/core/admin-core-api/src/test/java/org/codice/ddf/ui/admin/api/ConfigurationAdminTest.java

```diff
@@ -1208,10 +1210,12 @@ public void testGetClaimsConfiguration() {
         //check call before setting handler
         assertNotNull(configAdmin.getClaimsConfiguration(TEST_FILTER_1));
 
-        GuestClaimsHandlerExt ache = new GuestClaimsHandlerExt();
-        ache.setAvailableClaimsFile("/this/path/is/a/file");
-        ache.setProfileDir("/this/path/is/a/dir");
-        ache.setImmutableClaims("testClaim1,testClaim2");
+        GuestClaimsHandlerExt ache = new GuestClaimsHandlerExt(mock(PropertiesFileSupport.class),
+                Arrays.asList("testClaim1", "testClaim2"),
+                "/this/path/is/a/dir",
+                "/this/path/is/a/dir/banners",
+                "/this/path/is/a/file");
+
```

Comment:
Are we all ok with this? I think it's ok to _not_ add missing unit tests for _existing_ code or to refactor the tests after the code has been refactored, but we should add tests for the new logic we add.